### ISSUE TITLE
Help with installation 

### ISSUE THREAD

**Issue by amartirosyan** [MichaelB2018/lc7001-mqtt-bridge#1]
@MichaelB2018 I just started using HA and love to test and bring my lc7001 into HA as well, if you don't mind sharing how I can import this to HA?

**Comment by MichaelB2018** [comment]
Should be easy. Download the two files: lc7001-mqtt-bridge.py  & lc7001-mqtt-bridge.conf into the same folder. Update the MQTT connection parameters in lc7001-mqtt-bridge.conf. Then simply run lc7001-mqtt-bridge.py. It will connect to the LC7001 by itself and publish self discovery messages to MQTT which Home Assistant will read. it may taker about 2 minutes for 50 switches. But afterwards you will should see the switches on the Developer Tab of Home Assistant and you can include them on the LoveLace GUI too.

**Comment by amartirosyan** [comment]
Does it matter what folder on HA?

**Comment by MichaelB2018** [comment]
I'm running it on a separate RPi. But I'm quite sure you can run it on HA itself. no specific folder. should work as long as both are in the same folder

**Comment by amartirosyan** [comment]
@MichaelB2018 I assume, this is run once, or should I have dedicated host for it?

I am running HASOS and dont have access to the core, only basic SSH and the GUI

**Comment by MichaelB2018** [comment]
No, I didn't integrate it as such into HA. It's a standalone process that needs to run on an ongoing basis.

**Comment by merkurmaniac** [comment]
So a couple of basic questions.  
1. Does this permit me to eventually command the Adorne switches without even having the LC7001 plugged in, or is it still relaying commands thru that unit ?
2. What do I need to do to have it always running ?  Does it need to run in a docker, or how can I add it to HA ?  I am using the home assistant BLUE computer if that tells you anything about my hardware.
Thanks so much,
Richard

**Comment by MichaelB2018** [comment]
1.) you still need the LC7001. The Raspberry Pi by itself does not have RF capabilities.
2.) You can run it in docker. Or you can just start it up and it will run by itself. I'm running it on a standalone Raspberry Pi, but I suspect you can get it to run on BLUE as well.

**Comment by merkurmaniac** [comment]
1.  I do have a bond bridge if that would help me out.  I am using it to control a ceiling fan currently.  (if the LC1007 is an RF device, can something like the bond "sniff" the transmission and replicate it ?)
2. OK, another stupid question.  Is there the equivalent of an autoexec.bat that I can add your routine to so that it will run each time HA is rebooted ?

**Comment by MichaelB2018** [comment]
1.) Would be great if somebody could help with this!
2.) use crontab. I Just added the below line to my crontab as root ("sudo crontab"):
`@reboot sleep 60;sudo --user=pi /home/pi/lc7001-mqtt-bridge/lc7001-mqtt-bridge.py
`

**Comment by merkurmaniac** [comment]
Thank you.  I'll try the command line and see how it works out.

I'll also keep you posted if I find anything using the bond bridge.  I'll be out of pocket for about a week, unfortunately.

Thanks again for your work with these.  I love the Adorn switches and wall plates.

**Comment by 3DGenius79** [comment]
Hi [MichaelB2018]
Do you Speak German?
